fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -106,9 +106,9 @@ private void exportBinary(final URI uri)
             writeResponse(response, file);
         }
     }
-    private void exportContainer(final URI uri)
+    private void exportDescription(final URI uri, final URI... describes)
             throws FcrepoOperationFailedException, IOException {
-        final File file = fileForContainer(uri);
+        final File file = fileForContainer(describes.length > 0 ? describes[0] : uri);
```

Comment:
I wouldn't be hard to handle multiple descriptions, so I'll go ahead and add support for that.